src/revChatGPT/__main__.py

```diff
@@ -22,15 +25,17 @@ def get_input(prompt):
 
 
 if __name__ == "__main__":
-    print("""
+    print(
+        """
     ChatGPT - A command-line interface to OpenAI's ChatGPT (https://chat.openai.com/chat)
     Repo: github.com/acheong08/ChatGPT
-    """)
+    """,
+    )
     print("Type '!help' to show commands")
     print("Press enter twice to submit your question.\n")
 
     if exists("config.json"):
-        with open("config.json", "r", encoding='utf-8') as f:
+        with open("config.json", encoding="utf-8") as f:
```

Comment:
Is it not better to keep the 'r' to make sure it's only reading from it?

Reply:
I am going to fine tune this I apologize. You keep studying for exams and I will post a comment once ready. I hate to distract you from that. I will be working on this for a bit this evening.

Reply:
> You keep studying for exams

I just completed the last exam for this week a few minutes ago. I'm free to work on this now (though there are still internal assessments coming up. I will probably do those with ChatGPT as an assistant)

Reply:
I have narrowed this down to be a accurate fix as it was formatted under autopep8. I am testing now to see what rule of pep8 it is formatting to correct and comply with.